File: app/src/main/java/com/kamikaze/yada/Diary.java
package com.kamikaze.yada;

import android.location.Location;
import android.location.LocationManager;

public class Diary {
    private String title;
    private String description;
    public Diary(String title, String description) {
        this.title = title;
        this.description = description;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }
}

File: app/src/main/java/com/kamikaze/yada/diary/Diary.kt
package com.kamikaze.yada.diary

import com.kamikaze.yada.model.Notes

data class Diary(
    var title: String?,
    var description: String?,
    var location: String? = null,
    var bgImageUrl: String? = null,
    var note: Notes? = null,
    var color: Int = -1,
    var images: MutableList<String?>? = mutableListOf()
)